rdr_service/genomic/genomic_queries.py

```diff
@@ -1,481 +0,0 @@
298669202,298721343,300476476,303970614,304225283,304840344,307624541,311098094,311337192,316097003,317679728,321351007,326889159,328230727,328601272,329020340,330162708,331451633,336078084,336838765,337578880,337818631,338277574,338831211,340458347,341698344,341988377,343012588,343435863,345258636,345280414,348012855,349620163,350912055,353773042,353848033,355660493,357058315,366857236,367705012,367939265,368240133,371315218,373114023,373162526,376781979,382605411,387230641,394651840,395735978,397977638,401194112,401424274,402104802,404304180,406912635,408027090,412250175,413496554,431396034,436809693,445162321,447932598,448037661,449361516,451048031,458280644,458515650,458681939,465592827,467657277,468130853,468961337,469269759,470528358,481461052,484016469,489283953,491462847,491888224,496637872,497888796,505750540,514445174,515661072,529646079,531969575,533267737,533379876,537803205,539806802,545941327,548196679,551747640,553252185,555891024,557033065,557377304,558439473,561776629,563367445,563802897,566767416,568390849,568735259,570490060,571815053,577553524,579658179,587057490,592555841,593812228,594546037,600807440,601640155,602227169,604041895,607156896,613594760,617641484,619814680,625366140,626172751,627028164,630926453,634835987,639809016,641008375,642879319,643252934,643373675,645025662,648826157,650105369,651798663,654420029,659112961,659376061,665004584,670063147,672914529,673004218,677456779,678068632,685327852,687124540,689461120,700970968,701990050,705135637,705468325,709734671,710588213,714707946,721905336,725179865,725207446,727268499,727591271,729208904,737668017,738372198,740851280,745773279,746777052,747474400,759931867,760205595,761761517,763410162,767017612,768381785,779730744,779949118,785952569,787926464,791219357,794811079,795578393,797279247,810288873,812524724,814899961,819265696,819950283,820073586,823758344,830302099,831226243,836837874,837721794,838889749,841403776,844613792,848159433,851224539,851621512,852294303,853072726,853712780,855034665,857694544,858161509,859351011,859568629,865187888,865784343,866140322,868852263,869844960,877677445,879078463,880167360,880568948,884161802,885461547,893773868,894252703,895672943,905449614,907123717,907405741,910111063,913827794,914200698,914732151,918627535,918911212,920156311,921352328,922432405,924531063,924542857,926555538,926818829,926876459,927094113,933731079,945880676,950456601,951122259,953465025,955165843,955764343,962466291,966404836,973375903,974122861,979468523,979907745,984560898,985298078,985584918,985624484,987888386,989747214,990592038,990715447,993699312,994927821,995295154,995686976,999527800,999988289,116826403,148469844,186903103,228059843,298721343,311337192,329020340,336078084,338831211,367705012,382605411,387230641,413496554,447932598,468961337,481461052,489283953,533267737,568390849,645025662,672914529,837721794,838889749,865187888,866140322,924531063,926818829,955764343,270776467,295092346,303973848,322963871,593191281,696155559,705391345,711822745)
```

Comment:
There was a massive AW0 generation that happened a few months ago and these were participants that were flagged from research into the cohort discrepancies as not being, at the time, correctly defined as to what cohort group they were actually supposed to be in. So since this manifest generation/query was run, most of these participants have been correctly assigned to their cohort group, so the list is pretty much out of date at this time.

Reply:
Should the list be updated rather than removed? Are there any that should continue to be filtered while we still track down what cohort they should be a part of?

Reply:
Don't think so. The large manifest generation was "suppose" to a one-time run that the Biobank asked for and I think it's better to remove it from the code for readability and let it be resurrected if need be, which is a very slim chance, from git.